go-router/main/default/pages/go.page

```diff
@@ -18,11 +18,9 @@
         <apex:pageBlockTable value="{!routeGroup.routes}" var="route">
           <apex:column headerValue="Key" value="{!route.key}" width="15%" />
           <apex:column headerValue="Input" value="{!route.input}" width="35%" />
-          <apex:column
-            headerValue="Description"
-            value="{!route.usage}"
-            width="50%"
-          />
+          <apex:column headerValue="Description" width="50%" >
+            <apex:outputText value="{!route.usage}" escape="false"/>
```

Comment:
Does the open the link in a new tab in the browser or a separate tab within the Salesforce page? Which behavior would be better?

Reply:
Neither of the above, it opens as a static page not treated as a tab.